### PR TITLE
docs: Clarify Logits Processor and TRT-LLM examples

### PR DESCRIPTION
**Pull request by oandreeva-nv**
Renamed `logits_processor` parameter in the tutorial, to avoid confusion. 
Clarified `world_size` flag

### CODE REVIEW

File: Popular_Models_Guide/Hermes-2-Pro-Llama-3-8B/README.md [hunk 1]
```diff
@@ -174,6 +174,13 @@ python3 ${FILL_TEMPLATE_SCRIPT} -i ${MODEL_FOLDER}/tensorrt_llm/config.pbtxt tri
 
 3.  Launch Tritonserver
 
+> [!NOTE]
+> This tutorial was prepared for serving TensorRT-LLM model on a single GPU.
+> Thus, in the following command use `--world_size=1`, if you also built
+> an engine for a single GPU. Alternatively, if the engine requires multi-GPU
+> scenario, make sure to specify the exact number of GPU, required by you engine
+> in `--world_size`.
```
Comment (rmccorm4) [inline Popular_Models_Guide/Hermes-2-Pro-Llama-3-8B/README.md:182]: wording tweak
```suggestion
> This tutorial was prepared for serving a TensorRT-LLM model on a single GPU.
> Thus, in the following command use `--world_size=1` if the engine was built
> for a single GPU. Alternatively, if the engine requires multiple GPUs
> make sure to specify the exact number of GPUs required by the engine
> in `--world_size`.
```

**Review by rmccorm4 — APPROVED** [review]

File: AI_Agents_Guide/Constrained_Decoding/README.md
Comment (rmccorm4) [inline AI_Agents_Guide/Constrained_Decoding/README.md:1]: Suggested PR title rename, but feel free to use whatever you like
